Common/Models/PumpManagerUI.swift

```diff
@@ -9,9 +9,8 @@
 import Foundation
 import LoopKit
 import LoopKitUI
-import MockKitUI
 
-private let managersByIdentifier: [String: PumpManagerUI.Type] = staticPumpManagers.compactMap{ $0 as? PumpManagerUI.Type}.reduce(into: [:]) { (map, Type) in
+private let managersByIdentifier: [String: PumpManagerUI.Type] = staticPumpManagers.compactMap{ $0 }.reduce(into: [:]) { (map, Type) in
```

Comment:
[nit] Technically, this is a duplicate of `staticPumpManagersByIdentifier`. Consider removing. (Plus, I don't think it is actually used, but not sure.)